languages/csharp/exercises/concept/nullability/.docs/introduction.md

```diff
@@ -0,0 +1,102 @@
+Sometimes, we need to indicate that a variable has no value.  In C#,
+we use the `null` literal to denote the absence of a value.  A
+*nullable* type is a type that allows `null` values.  The operator `?`
+can be added as a suffix to a type to make it nullable.
+
+In the following example, we declare a regular, non-nullable variable
+`a` of type `int` and a nullable variable `b` of type `int?` whose
+value is `5`:
+
+```csharp
+int a = 4;
+int? b = 5;
+```
+
+`a` must contain an integer value, while `b` can also be set to
+`null`:
+
+```csharp
+a = null; // Compile error as a is not nullable
+a = 5;    // Valid
+
+b = null; // Valid
+b = 6;    // Valid
+```
```

Comment:
I think we can simplify this quite a bit by using comments:

```suggestion
```csharp
int? nullable = 5;
nullable = null;

int nonNullable = 5;

// Would result in compile error as type is not nullable
// nonNullable = null;
```